e2e/suites/actions-available/files-folders/files-folders-actions.test.ts

```diff
@@ -63,11 +63,15 @@ describe('Files / folders actions : ', () => {
 
   const loginPage = new LoginPage();
 
+  let initialSearchTotalItems: number;
+
   beforeAll(async () => {
     await adminApiActions.createUser({ username });
 
     parentId = (await userApi.nodes.createFolder(parent)).entry.id;
 
+    initialSearchTotalItems = await userApi.search.getTotalItems(username);
```

Comment:
not used outside the scope, so should be declared locally
```suggestion
    const initialSearchTotalItems = await userApi.search.getTotalItems(username);
```